### DISCUSSION TITLE
Max size reached

### DISCUSSION THREAD

**Discussion by KraigTersigni** [Security-Onion-Solutions/securityonion#4666] [Unsupported Versions] [answered]
We currently have version 2.3.51 deployed and when we export a csv from Elastic we get a Max size reached message. Which basically means the maximum file size we can download is 10 MB. This only allows us to download a couple hours worth of logs. I've gone through some configuration changes on the ./SecurityOnion/salt/kibana/etc/kibana.yml file to include the following:
xpack.reporting.csv.maxSizeBytes: 'some size bigger than default' but it doesn't seem to allow anything bigger than 10 MB to be exported. 

I've also added the above line to ./opt/so/conf/kibana/etc/kibana.yml but it gets wiped out after a short period of time. I suspected it had something to do with the above kibana.yml overwriting, but it doesn't appear to be the case. With that, has anyone been successful in exported a csv file larger than 10 MB?

**Comment by golfreeze** [comment]
found this error also try to change as 
xpack.reporting.csv.maxSizeByte: 20971520
on 
/opt/so/saltstack/default/salt/kibana/etc/kibana.yml
then could not access kibana 

Please kindly help T_T

**Comment by golfreeze** [comment]
after debug on docker logs so-kibana found error below on so 2.3.80 
 FATAL  Error: [config validation of [xpack.reporting].csv.maxsizebyte]: definition for this key is missing

**Comment by golfreeze** [comment, marked as answer]
After checking add by Byte(s) as below 
xpack.reporting.csv.maxSizeBytes: 20971520
on
/opt/so/saltstack/default/salt/kibana/etc/kibana.yml
and then 
#so-kibana-restart 
could working normally.
